bears/python/PythonPackageInitBear.py

```diff
@@ -0,0 +1,35 @@
+import os
+import re
+
+from coalib.results.Result import Result
+from coalib.bears.GlobalBear import GlobalBear
+
+
+class PythonPackageInitBear(GlobalBear):
+    """
+    Looks for missing __init__.py files in directories containing python files.
+    """
+
+    def run(self,
+            root_directory: str = '.',
+            ignore_dirs: list = None,
+            create_empty_init_file: bool = True):
+        """
+        :param root_directory:
+            The root directory of the project whose tree has to be checked
+        :ignore_dirs:
+            List of directories to be ignored
+        """
+
+        for dirpath, dirs, files in os.walk(root_directory):
```

Comment:
AFAIK the dict contains filenames, I can't get its directory, nor can I search the directory for init file that way.